scripts/test_copy_chunk_data.sh

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env bash
+
+#
+# Test copy_chunk_data(): using logical replication to copy chunk data
+# between data nodes
```

Comment:
Any chance we could utilize TAP tests in this one? I realize this is additional and probably unnecessary work and don't insist on it. On the other hand, Perl code is much easier to maintain than Bash and would be nice to be consistent with future tests.